sqlitecluster/SQLiteNode.cpp

```diff
@@ -1635,11 +1643,6 @@ void SQLiteNode::_onMESSAGE(SQLitePeer* peer, const SData& message) {
                       << message.calc("NewCount") << " (" << message["NewHash"] << ", " << message["ID"] << ") but '"
                       << e.what() << "', ignoring.");
             }
-        } else if (SIEquals(message.methodLine, "CRASH_COMMAND") || SIEquals(message.methodLine, "BROADCAST_COMMAND")) {
-            // Create a new Command and send to the server.
-            SData messageCopy = message;
-            PINFO("Received " << message.methodLine << " command, forwarding to server.");
-            _server.acceptCommand(make_unique<SQLiteCommand>(move(messageCopy)), true);
```

Comment:
This is because we stopped sending these this way in https://github.com/Expensify/Bedrock/pull/1303, right?